### DISCUSSION TITLE
Problem Updating to Elsa 2.3 from 2.1

### DISCUSSION THREAD

**Discussion by Alexplose** [elsa-workflows/elsa-core#1591] [Q&A]
I get this error since I updated : 
`System.InvalidOperationException: 'Cannot resolve 'Microsoft.AspNetCore.Authorization.IAuthorizationService' from root provider because it requires scoped service 'Microsoft.AspNetCore.Authorization.IAuthorizationHandler'.'
`
Any help appreciated

**Comment by sfmskywalker** [comment]
I'll push an update that fixes that, but until then you should be able to fix it by adding the following code to your Startup class:

```csharp
services.AddAuthorization();
```

**Reply by noxe** [reply]
Having the same problem - but the workaround does not work. unfortunately i cannot isolate it in a small sample currently

**Comment by akghasemi** [comment]
I have the same problem so I opened a new issue and had to use version 2.2.1

**Comment by MehdiSaeedifar** [comment]
I have the same issue. Any Update?

**Reply by sfmskywalker** [reply]
I haven’t been able to reproduce this issue. If someone sends me a (stripped down) project I can look at I might see what’s going on.
